get_client_token.py

```diff
@@ -0,0 +1,34 @@
+def get_client_token():
+    from requests.auth import HTTPBasicAuth
```

Comment:
to przed funkcję, nie lubię importów wewnątrz funkcji bo potem jak będziesz chciał odświeżyć token na nowo wszystko będzie zaciągać